package/pyproject.toml

```diff
@@ -118,3 +118,8 @@ find = {}
 MDAnalysis = [
     'analysis/data/*.npy',
 ]
+
+[tool.black]
+line-length = 79
```

Comment:
We need to decide if we really want to stick to a line length of `79` as expressed in the UserGuide or use `black`'s default of `88`.

Reply:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Reply:
@IAlibay @PicoCentauri thoughts?

Reply:
79, let's pin to one of the standards (I think you can to the right), and otherwise lgtm!

Reply:
Tested a pin to 23 with black 24:
```
⏰ 22:39:00 ☑️   black package
Oh no! 💥 💔 💥 The required version `23` does not match the running version `24.8.0`!
```